types/chrome/index.d.ts

```diff
@@ -4205,6 +4205,20 @@ declare namespace chrome.input.ime {
      * function(boolean success) {...};
      */
     export function setAssistiveWindowProperties(parameters: object, callback?: (success: boolean) => void): void;
+    /**
+     * Highlights/Unhighlights a button in an assistive window.
+     * @param {{
+     *   contextID: number,
+     *   buttonID: AssistiveWindowButton,
+     *   windowType: AssistiveWindowType,
+     *   announceString?: string,
+     *   highligted: boolean
+     * }} parameters
+     * @param callback Called when the operation completes. On failure, chrome.runtime.lastError is set.
+     * If you specify the callback parameter, it should be a function that looks like this:
+     * function() {...};
+     */
+    export function setAssistiveWindowButtonHighlighted(parameters: object, callback?: (success: boolean) => void): void;
```

Comment:
1. You can give more details to `parameters` type.
2. `AssistiveWindowButton` & `AssistiveWindowType` are already defined as types. Please use them.

like this
```ts
export function setAssistiveWindowButtonHighlighted(
  parameters: { contextID: number, buttonID: AssistiveWindowButton, ...},
  ...
```

3. The `callback` seems to take no arguments.
https://developer.chrome.com/extensions/input_ime#method-setAssistiveWindowButtonHighlighted

Reply:
Hi pokutuna, I have updated that parameters. Please take a look 👍